docs/checks.md

```diff
@@ -12,13 +12,12 @@ If your PMM instance is not connected to Percona Platform, PMM can only download
 As soon as you connect your PMM instance to Percona Platform, has access to additional checks, available only for Registered PMM instances.
 
 If you are a Percona customer with a Percona Customer Portal account, you also get access to Paid checks, which offer more advanced database health information.
-​
 
 ## Checks results
 
-The checks can be executed manually or automatically. By default, PMM runs automatic checks every 24 hours.
+The checks can be executed manually or automatically. Manual checks can be executed individually, or all together. 
```

Comment:
Do we really want to remove that mention of how often we run it? Or is no longer true?

Reply:
@atymchuk yeah, the "24 hours" interval is only true for a subset of checks. We have three intervals with these defaults:
Rare -> 78 hours
Standard -> 24 hours
Frequent -> 4 hours

So how often we run a check depends on the interval for that check.